src/frontend/qt_sdl/Config.cpp

```diff
@@ -26,8 +26,8 @@
 namespace Config
 {
 
-int KeyMapping[12];
-int JoyMapping[12];
+int KeyMapping[14];
```

Comment:
why are you increasing this?

Reply:
There are now 14 hotkey variables for both keys and joypads.

Reply:
these arrays store the mapping for keyboard keys and gamepad to DS keys (ABXY, Dpad, L, R, Start, Select), not hotkeys.

The reason you changed this was probably, because before your change there were 12 hotkeys as well, though that's a pure coincidence.

The hotkey mapping are stored in HKKeyMapping and HKJoyMapping. They were automatically resized to 14 elements when you added elements to the enum, thanks HK_MAX moving further to the back.

Reply:
Ah okay. Makes sense. I can revise this.